### PR TITLE
Support primary_source

### PR DESCRIPTION
**Pull request by likhitha-surapaneni**
API fetched primary_source from the header previously. Modified logic to check from INFO column first and then fallback to fetching from the header.

### CODE REVIEW

File: common/file_model/variant.py [hunk 1]
```diff
@@ -46,27 +55,27 @@ def get_primary_source(self) -> Mapping:
...
                 source_url = "https://www.ncbi.nlm.nih.gov/clinvar/variation/"
                 source_release = ""
+            
+            elif re.search("^EVA", source):
+                source_id = "EVA"
+                source_name = "EVA"
+                source_description = "European Variation Archive"
+                source_url = "https://www.ebi.ac.uk/eva"
+                source_url_id = "https://www.ebi.ac.uk/eva/?variant&accessionID="
+                source_release = ""
```
Comment (nakib103) [inline common/file_model/variant.py:65]: We can also have "Ensembl" source

File: common/file_model/variant.py [hunk 2]
```diff
@@ -46,27 +55,27 @@ def get_primary_source(self) -> Mapping:
...
 
-        except:
+        except Exception as e:
             return None
```
Comment (nakib103) [inline common/file_model/variant.py:68]: If we cannot get any of them should we send None, primary_source being non-nullable.
`primary_source: ExternalReference!`

Maybe we can say something like `Unknown` or `Undeterminable`.

**Review by nakib103 — CHANGES_REQUESTED** [review]

File: common/file_model/variant.py [hunk 3]
```diff
@@ -31,8 +31,17 @@ def get_alternative_names(self) -> List:
...
+        Fallsback to fetching from the header
+        """
+        
         try:
-            source = self.header.get_lines("source")[0].value
+            if "SOURCE" in self.info:
+                source = self.info["SOURCE"]
+            else:
+                source = self.header.get_lines("source")[0].value
+
```
Comment (nakib103) [inline common/file_model/variant.py:44]: The DbSNP release is now `156`

**Review by nakib103 — CHANGES_REQUESTED** [review]

File: common/file_model/variant.py (hunk 2 above)
Comment (likhitha-surapaneni) [reply common/file_model/variant.py:75]: Currently, we are sending None as part of error handling forcing the api to fail when the source is not in the expected format. I agree. We should also handle sources other than dbSNP, EVA and Ensembl.

**Review by nakib103 — APPROVED** [review]
Looks Good!~

**Pull request merged**